src/services/auth-management/notifier.ts

```diff
@@ -0,0 +1,154 @@
+// Initializes the `authmanagement` service on path `/authmanagement`
+import { Application } from '../../declarations';
+import * as path from 'path';
+
+const pug = require('pug');
+
+export default function (app: Application) {
+  function getLink(type: string, hash: string) {
```

Comment:
Do you mind extracting `getLink` and `sendEmail` to their own standalone functions, just above? I'd like to avoid embedded functions if at all possible.